src/payment/strategies/paypal-commerce/paypal-commerce-payment-processor.ts

```diff
@@ -263,4 +289,25 @@ export default class PaypalCommercePaymentProcessor {
         throw new NotImplementedError(`PayPal ${this._fundingSource || ''} is not available for your region. Please use PayPal Checkout instead.`);
     }
 
+    private getOrderRequestBody(gatewayId: string | undefined, methodId: string, paymentData: PaymentInstrument & PaymentInstrumentMeta): OrderRequestBody {
+        return {
+            useStoreCredit: false,
+            payment: this.getPayment(gatewayId, methodId, paymentData) as OrderPaymentRequestBody,
+        };
+    }
+
+    private getPayment(gatewayId: string | undefined, methodId: string, paymentData: PaymentInstrument & PaymentInstrumentMeta): Payment {
+        return {
+            methodId,
+            gatewayId,
+            paymentData,
+        };
+    }
+
+    private getPaymentRequestOptions(gatewayId: string | undefined, methodId: string): PaymentRequestOptions {
+        return {
+            gatewayId,
+            methodId,
+        };
+    }
```

Comment:
The  `getOrderRequestBody`, `getPayment`, `getPaymentRequestOptions` methods can be removed